lib/modules/datasource/pypi/index.ts

```diff
@@ -83,21 +89,21 @@ export class PypiDatasource extends Datasource {
     return input.toLowerCase().replace(regEx(/(_|\.|-)+/g), '-');
   }
 
-  private async getDependency(
+  private async addResultsViaPyPiJson(
     packageName: string,
     hostUrl: string,
-  ): Promise<ReleaseResult | null> {
+    dependency: ReleaseResult,
```

Comment:
Return the release result rather than manipulating it in a private function intransparently.

Reply:
Thanks for your review! I've done so, so that the metadata can easily be added to existing results. Now, merging is done afterwards with a simple `Object.assign`.